storage_controller/src/service.rs

```diff
@@ -1159,6 +1153,16 @@ impl Service {
         let (result_tx, result_rx) = tokio::sync::mpsc::unbounded_channel();
         let (abort_tx, abort_rx) = tokio::sync::mpsc::unbounded_channel();
 
+        let leadership_cancel = CancellationToken::new();
+        let leadership = Leadership::new(persistence.clone(), config.clone(), leadership_cancel);
+        let (leader, leader_step_down_state) = leadership.prologue().await?;
```

Comment:
As a non-native english speaker my first idea is: why no epilogue?

I'd rather just use names like "become_leader" or something like that.

Reply:
Yeah... `prologue` basically means: "figure out who the leader is and tell them to step-down` and  `epilogue` means: "mark yourself as the leader". I'll rename them to something along those lines.

Reply:
Renamed in https://github.com/neondatabase/neon/pull/8756/commits/1df537e7ec943b096260742dc421faf19b039a48